src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/cars/add")
+public class CreateCarController extends HttpServlet {
```

Comment:
```suggestion
public class AddCarController extends HttpServlet {
```
if you use verb `add` in url, it's better to use the same verb in controller's name, fix it everywhere